client/src/components/jobPosting/JobPostingStages.js

```diff
@@ -75,41 +122,64 @@ export class JobPostingStages extends Component {
     const error = this.state.error
     const helperText = error ? 'Invalid stage name. Stage was not added' : this.state.helperText
 
+    const classNames = 'job-posting-form-stages'
     return (
-      <div className='job-posting-form-stages' onKeyPress={this.handleKeyPress}>
+      <div className={classNames}>
         <h3>Define stages for this job posting</h3>
-        <div className='job-posting-form-stages__new-stage-name'>
+        <div className={classNames + '__new-stage-name'}>
           <TextField
             fullWidth
             id="stageName"
             type="text"
             value={this.state.newStageName}
             label={`Add ${this.props.jobPostingStages.length - 1}. stage (optional)`}
             onChange={this.handleNameChange}
+            onKeyPress={this.handleKeyPress}
             variant="outlined"
             helperText={helperText}
             error={error}
           />
         </div>
-        <div className='job-posting-form-stages__new-stage-add-button'>
+        <div className={classNames + '__new-stage-add-button'}>
           <Button mini variant="fab" color="inherit" aria-label="Add" onClick={() => this.addNewStage()}>
             <AddIcon />
           </Button>
         </div>
-        <div className='job-posting-form-stages__stage-list'>
+        <div className={classNames + '__stage-list'}>
           {this.props.jobPostingStages.map((jobPostingStage, index) => (
-            <div key={index} className='job-posting-form-stages__single-stage'>
+            <div key={index} className={classNames + '__single-stage'}>
```

Comment:
Name is probably a better key than index especially if name is unique

Reply:
This comment is still valid